flask_caching/__init__.py

```diff
@@ -264,6 +264,13 @@ def get_dict(self, *args, **kwargs):
         """Proxy function for internal cache object."""
         return self.cache.get_dict(*args, **kwargs)
 
+    def unlink(self, *args, **kwargs):
+        """Proxy function for internal cache object
+        only support Redis
+        """
+        unlink = getattr(self.cache, "unlink", None)
+        return unlink(*args, **kwargs) if unlink is not None else None
```

Comment:
Since `unlink` is only supported by Redis, other backends (and so will older Redis versions) will default to a no-op here. You should either resort to a plain `delete()` operation if `unlink` is not available, or at least issue a warning that the backend doesn’t support this call.

Reply:
yep. I had the change which you said, but I'm not sure  it's good.